### PR TITLE
Running on smaug

### PR DESCRIPTION
**Pull request by CodyCBakerPhD**
@bendichter Small changes per slack discussion.

`run_conversion` also prints out name of currently running conversion files, and `convert_dir` skips previously written NWBFiles to save runtime and computations.

### CODE REVIEW

File: brunton_lab_to_nwb/brunton_lab_to_nwb.py [hunk 1]
```diff
@@ -197,8 +198,11 @@ def run_conversion(
 
 
 def convert_dir(in_dir, n_jobs=1):
-    in_files = glob(os.path.join(in_dir, '*.h5'))
-    out_files = [os.path.splitext(x)[0] + '.nwb' for x in in_files]
+    all_files = Path(in_dir).iterdir()
+    all_data_files = [x.stem for x in all_files if ".h5" in x.suffix]
+    nwb_files = [x.stem for x in all_files if ".nwb" in x.suffix]
+    in_files = [str(in_dir / f"{x}.h5") for x in all_data_files if x not in nwb_files]
+    out_files = [str(in_dir / f"{Path(x).stem}.nwb") for x in in_files]
```
Comment (bendichter) [inline brunton_lab_to_nwb/brunton_lab_to_nwb.py:205]: Would be better to have an `overwrite=True` option on this function

File: elec_loc_labels.csv [hunk 2]
```diff
@@ -1,4 +1,4 @@
-﻿subject_ID,label,long_name
+subject_ID,label,long_name
```
Comment (bendichter) [inline elec_loc_labels.csv:1]: Do you know what's going on here?

File: brunton_lab_to_nwb/brunton_lab_to_nwb.py (hunk 1 above)
Comment (CodyCBakerPhD) [reply brunton_lab_to_nwb/brunton_lab_to_nwb.py:205]: Sure, I'll add that

File: elec_loc_labels.csv (hunk 2 above)
Comment (CodyCBakerPhD) [reply elec_loc_labels.csv:1]: As in, why that's in the changelog even though nothing changed?

File: brunton_lab_to_nwb/brunton_lab_to_nwb.py [hunk 3]
```diff
@@ -197,8 +198,11 @@ def run_conversion(
...
 
     Parallel(n_jobs=n_jobs)(
```
Comment (bendichter) [inline brunton_lab_to_nwb/brunton_lab_to_nwb.py:207]: Thanks for parallelizing

Comment (bendichter) [reply brunton_lab_to_nwb/brunton_lab_to_nwb.py:207]: Could we also have an `n_jobs` default arg on this function?

File: elec_loc_labels.csv (hunk 2 above)
Comment (bendichter) [reply elec_loc_labels.csv:1]: yeah

Comment (CodyCBakerPhD) [reply elec_loc_labels.csv:1]: Not a clue; I just opened it in excel, added the line, and closed. Maybe a windows vs mac file encoding thing like EOF's or temp's sometimes are?

File: brunton_lab_to_nwb/brunton_lab_to_nwb.py (hunk 3 above)
Comment (CodyCBakerPhD) [reply brunton_lab_to_nwb/brunton_lab_to_nwb.py:207]: I'm not the one that added the parallelizing, lol, was that way when I cloned it. `n_jobs` currently defaults to `1`. (But I use `12` on smaug)

File: brunton_lab_to_nwb/brunton_lab_to_nwb.py (hunk 1 above)
Comment (CodyCBakerPhD) [reply brunton_lab_to_nwb/brunton_lab_to_nwb.py]: Added

File: elec_loc_labels.csv (hunk 2 above)
Comment (bendichter) [reply elec_loc_labels.csv:1]: yeah, probably something like that

**Pull request merged**